scijava-ops-engine/src/test/java/org/scijava/ops/engine/matcher/convert/ConversionTest.java

```diff
@@ -169,4 +171,43 @@ public void testConvertAnys() {
 		Assertions.assertInstanceOf(Integer.class, out);
 	}
 
+	/**
+	 * An Op, written as a method, whose type variable has multiple bounds.
+	 * <p>
+	 * Note that, for the purposes of this test, the {@link Number} bound is
+	 * necessary even though it is not needed for the functionality of the test
+	 * Op.
+	 * </p>
+	 */
+	@OpMethod(names = "test.boundsConversion", type = BiFunction.class)
```

Comment:
Once again I got confused by the name of this op. It's weird to me that we have test ops named after the *functionality being tested* rather than named for *what they do* like normal ops. In this case, the op `test.boundsConversion` is a static method named `foo` (which I find unhelpful in understanding) and it returns the max of two objects. So why not name it "max" instead? Especially since the functionality being tested here, multiple type bounds, is also a sort of comparison, which was the source of my momentary confusion: "What does `x > y ? x : y` have to do with *multiple* bounds?" I thought, "It's just *one* bound..."